### PR TITLE
tiltfile: test for top level for loop

### PR DESCRIPTION
**Pull request by jazzdan**

### CODE REVIEW

File: internal/tiltfile/tiltfile_test.go [hunk 1]
```diff
@@ -932,6 +932,20 @@ if True:
...
+	defer f.TearDown()
+
+	f.setupFoo()
+
+	f.file("Tiltfile", `
+for i in range(1, 3):
+	print(i)
+`)
+
+	f.load()
```
Comment (maiamcc) [inline internal/tiltfile/tiltfile_test.go]: this... doesn't test anything? are you just trying to make sure it doesn't crash?

Comment (jazzdan) [reply internal/tiltfile/tiltfile_test.go]: Yep!

Comment (maiamcc) [reply internal/tiltfile/tiltfile_test.go]: ...okay 🤷‍♀️ can you leave a comment or rename the test to `TestTopLevelForLoopDoesntCrash` or something?

Comment (jazzdan) [reply internal/tiltfile/tiltfile_test.go]: @maiamcc for context this isn't allowed normally in Starlark, and is behavior I [specifically enabled](https://github.com/windmilleng/tilt/blob/master/internal/tiltfile/tiltfile.go#L27) to allow top level if statements and I wanted to make sure that it also applied to the for loop control structure.

Comment (maiamcc) [reply internal/tiltfile/tiltfile_test.go]: Oooh i see! (Worth noting that in a comment)

**Pull request merged**